### ISSUE TITLE
Problem text doesn't specify text color

### ISSUE THREAD

**Issue by matthewmorgan** [FreeCodeCamp/FreeCodeCamp#5980]
Sounds trivial, but the problem text specifies a class name without specifying that the text *color* should be changed.  The class name does not make the color change, the distinction should be clear.

Challenge [Waypoint: Use a CSS Class to Style an Element](http://www.freecodecamp.com/challenges/waypoint-use-a-css-class-to-style-an-element#?solution=%3Cstyle%3E%0A%20%20.red-text%20%7B%0A%20%20%20%20color%3A%20blue%3B%0A%20%20%7D%0A%3C%2Fstyle%3E%0A%0A%3Ch2%20class%3D%22red-text%22%3ECatPhotoApp%3C%2Fh2%3E%0A%0A%3Cp%3EKitty%20ipsum%20dolor%20sit%20amet%2C%20shed%20everywhere%20shed%20everywhere%20stretching%20attack%20your%20ankles%20chase%20the%20red%20dot%2C%20hairball%20run%20catnip%20eat%20the%20grass%20sniff.%3C%2Fp%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html
<style>
  .red-text {
    color: blue;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>

```

**Comment by TravisHi** [comment]
Consider changing "Create a CSS class called red-text and apply it to your h2 element.", to

"Create a CSS class called red-text and apply it to your h2 element. Remember to change `color:` to red in your 'red-text' class."

**Comment by matthewmorgan** [comment]
I see.  Not in the main problem text, but in the bullet points below.  My oversight-- sorry!

**Comment by sludge256** [comment]
It should still be in the instructions though. You should leave this issue open for now.

**Comment by TravisHi** [comment]
Yeah, wouldn't hurt to have clear instructions.

**Comment by SaintPeter** [comment]
> Create a CSS class called red-text and apply it to your h2 element. Change the `color` to `red` in your 'red-text' class.

Seems reasonable to me.

**Comment by 0x0936** [comment]
I've this waypoint show up fairly often in the help channel, and I agree the instructions may need to be refreshed for more clarity.

### Current Instructions

> Instead of creating a new style element, try removing the h2 style declaration from your existing style element, then replace it with the class declaration for .red-text
>
> Create a CSS class called red-text and apply it to your h2 element.

### Proposed Change

> Inside your `style` element, change the `h2` selector to `.red-text` and update the color's value from blue to red.
>
> Give your `h2` element a `class` attribute with the value `'red-text'`

Terminology consistent with the W3

![img](http://www.w3.org/Style/LieBos2e/enter/anatomy-diag.png)

**Comment by matthewmorgan** [comment]
To be clear, I know how to change the text color.  What wasn't clear was that it was expected in the exercise.  Embarrassingly, this is because I didn't even scroll down in the browser to look at the bullet points.  I didn't question why the style name was changed without changing the actual color, I guess I thought it might be an intermediate step in a longer sequence of tasks.

Until I ran in to this snag, I never expanded my browser to show the bullet points at the bottom.
